libraries/AP_InertialSensor/AP_InertialSensor_MPU9250.cpp

```diff
@@ -438,7 +438,7 @@ bool AP_InertialSensor_MPU9250::_accumulate(uint8_t *samples, uint8_t n_samples)
             _fifo_reset();
             return false;
         }
-        float temp = t2/340 + 36.53;
```

Comment:
there's been a bit of back and forth about casting using C++ method instead of C. Do we have an official stance on that? If so, we should convert the whole project to C++